app/js/app.js

```diff
@@ -1,15 +1,15 @@
 var conceptDictionary = angular.module('conceptDictionary', 
-		['ngRoute', 'conceptDictControllers', 'conceptDictServices']);
+		['ngRoute', 'conceptDictControllers', 'conceptDictServices', 'openmrs.resources']);
 
 conceptDictionary.config(['$routeProvider',
-                    function($routeProvider) {
+                    function($routeProvider, openmrsRest) {
                       $routeProvider.
                         when('/class-list', {
                           templateUrl: 'partials/class-list.html',
                           controller: 'ClassesListCtrl',
                           resolve: {
-                        	  loadClasses : function(ClassesService){
-                        		  return ClassesService.getAll();
+                        	  loadClasses : function(openmrsRest){
+                        		  return openmrsRest.listFull('conceptclass', {});
```

Comment:
You need to correct the listFull implementation so that it doesn't extend the query rather creates a new one when nothing is passed.